### PR TITLE
[GETUNAME] Update Polish (pl-PL) translation

### PR DESCRIPTION
**Pull request by pithwz**
[GETUNAME] Update Polish (pl-PL) translation

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-polish for review

**Review by archeYR — CHANGES_REQUESTED** [review]
Please revert the line ending changes.

**Comment by binarymaster** [comment]
> Please revert the line ending changes.

Fixed it. Perhaps the file was edited under Linux...

File: dll/win32/getuname/lang/pl-PL.rc [hunk 1]
```diff
@@ -167,38 +169,38 @@ BEGIN
...
+    171 "Lewy podwójny cudzysłów kątowy"
     172 "Not Sign"
-    173 "Soft Hyphen"
-    174 "Registered Sign"
+    173 "Łącznik nietrwały"
+    174 "Zastrzeżony znak towarowy"
     175 "Macron"
-    176 "Degree Sign"
-    177 "Plus-Minus Sign"
+    176 "Zmak stopnia"
```
Comment (archeYR) [inline dll/win32/getuname/lang/pl-PL.rc:188]:
```suggestion
    176 "Znak stopnia"
```

File: dll/win32/getuname/lang/pl-PL.rc [hunk 2]
```diff
@@ -8737,35 +8739,35 @@ BEGIN
...
     8738 "Spherical Angle"
     8739 "Divides"
     8740 "Does Not Divide"
     8741 "Parallel To"
     8742 "Not Parallel To"
-    8743 "Logical And"
-    8744 "Logical Or"
-    8745 "Intersection"
-    8746 "Union"
+    8743 "Koniunkcja And"
```
Comment (archeYR) [inline dll/win32/getuname/lang/pl-PL.rc:8755]:
```suggestion
    8743 "Koniunkcja"
```

File: dll/win32/getuname/lang/pl-PL.rc [hunk 3]
```diff
@@ -8737,35 +8739,35 @@ BEGIN
...
+    8744 "Alternatywa Or"
```
Comment (archeYR) [inline dll/win32/getuname/lang/pl-PL.rc:8756]:
```suggestion
    8744 "Alternatywa"
```

File: dll/win32/getuname/lang/pl-PL.rc [hunk 4]
```diff
@@ -8737,35 +8739,35 @@ BEGIN
...
+    8745 "Iloczyn zbiorów"
+    8746 "Składająca"
```
Comment (archeYR) [inline dll/win32/getuname/lang/pl-PL.rc:8758]:
```suggestion
    8746 "Suma zbiorów"
```

**Review by archeYR — CHANGES_REQUESTED** [review]

**Review by archeYR — APPROVED** [review]

**Pull request merged**